src/Extend/UserPreferences.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Extend;
+
+use Flarum\Extension\Extension;
+use Flarum\User\User;
+use Illuminate\Contracts\Container\Container;
+
+class UserPreferences implements ExtenderInterface
```

Comment:
Should we have a method for changing the default value/transformer of a preference? Or perhaps that can come after we figure out extension boot order.